src/classes/header-surrogate-control.php

```diff
@@ -61,5 +61,6 @@ public function get_seconds() {
 	 */
 	public function set_seconds( $seconds ) {
 		$this->_seconds = $seconds;
+		$this->set_value( $this->prepare_value( $seconds ) );
```

Comment:
I noticed the `purgely_set_ttl` helper function was also broken due to this missing